federation/schema.go

```diff
@@ -115,6 +115,14 @@ func ConvertVersionedSchemas(schemas serviceSchemas) (*SchemaWithFederationInfo,
 						if !ok {
 							return nil, oops.Errorf("Object %s is not an input object, but it is an argument to the field %s", rootType.Name, field.Name)
 						}
+
+						// Check that all the input fields are on the federated object
+						for fName := range inputType.InputFields {
+							if _, ok := inputType.InputFields[fName]; !ok {
```

Comment:
`if _, ok != obj.Fields[fName]`? 
Do you think we should add an unit test for it?

Reply:
updated and added a unit test